tests/testing/ava/fixture.test.ts

```diff
@@ -0,0 +1,14 @@
+import test from "ava"
+import { fileURLToPath } from "node:url"
+import path from "node:path"
+import { getTestServer } from "src/testing/ava/fixture"
+
+test("AVA fixture works", async (t) => {
+  const { port } = await getTestServer(t, {
+    rootDirectory: path.dirname(fileURLToPath(import.meta.url)),
+  })
```

Comment:
if reviewing, start here: this is how consumers use the fixture

Reply:
woops, i ended here 😅